### DISCUSSION TITLE
Mi proyecto con la extensión de VS Live Server no se ve igual que cuando lo abro localmente

### DISCUSSION THREAD

**Discussion by victor-gp99** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#411] [Q&A] [answered]
Hola que tal compañeros tengo una duda, ¿porque cuando abro mi proyecto con visual studio code con la extencion se live server se ve bien? como le he estado programando pero cuando abro el archivo .html no se ve igual que en el live server...
![conLiveServer](https://user-images.githubusercontent.com/58193171/158305417-d5bfd012-2c4c-486c-a0b7-d63ef7144f01.PNG)
![local](https://user-images.githubusercontent.com/58193171/158305419-f38c78ec-e21f-4ec0-ad58-77ccf9477ef7.PNG)

**Comment by alanglez45** [comment]
Es normal, al momento de desplegarla en un servidor se verá como la ves con live server. Don't worry

**Reply by victor-gp99** [reply]
Gracias hermano :)

**Comment by hackit92** [comment]
Es muy curioso, habría que ver que no está cargando el código, pero me suena a que estás declarando tus rutas como absolutas y no relativas

**Reply by victor-gp99** [reply]
Lo tendré en cuenta gracias :)

**Comment by doguedogue** [comment, marked as answer]
trata de usar la ruta de las imágenes empezando con punto + diagonal    ./ruta_imagen/imagen.png
por eso no se te ven cuando abres el html como archivo

**Reply by victor-gp99** [reply]
Intentare tu consejo hermano gracias:)
